spark/src/main/kotlin/com/adevinta/spark/components/toggles/CheckBox.md

```diff
@@ -0,0 +1,151 @@
+# CheckBox components
+
+## CheckBox design specs
+
+You can find the design specs
+on [zeroheight.com/25c15666f/](https://zeroheight.com/25c15666f/p/72b9ad-checkbox-/b/057def).
+
+## Usage
+
+### CheckBox
+
+| Light                                                                                                                                                                      | Dark                                                                                                                                                                      |
+|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ![](../../../../../../../../../spark-screenshot-testing/src/test/snapshots/images/com.adevinta.spark_PreviewScreenshotTests_preview_tests_toggles_checkbox_part_light.png) | ![](../../../../../../../../../spark-screenshot-testing/src/test/snapshots/images/com.adevinta.spark_PreviewScreenshotTests_preview_tests_toggles_checkbox_part_dark.png) |
```

Comment:
In the future, I think we should extract all these inline links into reference links at the end of the page.

Reply:
Yeah it'll be way more readable 👍